src/quo2/components/settings/privacy/option.cljs

```diff
@@ -0,0 +1,63 @@
+(ns quo2.components.settings.privacy.option
```

Comment:
hi @gdevincenzi,
Please can you rebase the PR and also, please use the same directory structure for the component as of the preview screen (and designs).
`quo2.components.settings.privacy-options`

Reply:
Should be all good now